c/protocol.h

```diff
@@ -0,0 +1,1864 @@
+// Generated by Molecule 0.4.1
```

Comment:
Why not reference this file from `deps/ckb-c-stdlib`?

Reply:
Thank you for spotting this. I updated my code to use `blockchain.h` from `deps/ckb-c-stdlib`.

Reply:
And also need remove `protocol.h`.

Reply:
Roger that. Thanks.